### PR TITLE
Fixes from weekly meeting

### PR DESCRIPTION
**Pull request by jukent**
- uses truncated color map images for packages
 - removes maintenance code individual pages and shortens section
 - removes tutorials page
 - makes banners unique for each page
 - makes font styles consistent for each page
 - removes unused code

### CODE REVIEW

File: themes/ncar/layouts/software/list.html [hunk 1]
```diff
@@ -66,27 +66,19 @@ <h3>{{ .package }}</h3>
...
-    <h2>
-      Under Maintenance
-    </h2>
-    <p style="color: black;">Blurb describing what we mean by maintenance software.</p>
-    <div class="row">
+    <h2>Under Maintenance</h2>
+    <p style="color: black;">
+      NCAR has made the decision to adopt Python as the scripting language platform of choice for future development of analysis and visualization tools. 
+      As part of this decision, NCL, PyNGL, and PyNIO have been placed into maintenance mode. 
+      Please read this <a href="https://www.ncl.ucar.edu/open_letter_to_ncl_users.shtml">open letter to NCL users</a> to understand what kind of impact this will have on the future of these projects.
```
Comment (kafitzgerald) [inline themes/ncar/layouts/software/list.html:73]: Maybe we just leave this as is for now, but I think we'll want to either update this page and/or point to something with more recent information.  

I still need to make some updates to those older NCL / PyNIO / PyNGL pages and I think the best approach might depend on what those look like. 

How about I log an issue on here as a placeholder and target getting done in the next week or so?

Comment (jukent) [reply themes/ncar/layouts/software/list.html:73]: Sounds great

**Review by kafitzgerald — APPROVED** [review]
I didn't go through build this to preview it, but did look through the diffs and it all seems reasonable to me as far as I can tell.

Thanks for getting this all updated!

**Comment by jukent** [comment]
Thanks @kafitzgerald I'll open an issue for each of the remaining to do items

**Pull request merged**